packages/itwin/ec3-widget/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "./node_modules/@itwin/build-tools/tsconfig-base.json",
+  "compilerOptions": {
+    "skipLibCheck": true,

```

Comment:
`skipLibCheck` and `jsx` match what is in the `tsconfig-base.json` file, to they can be removed here. Could you explain why you need `preserveSymlinks` and `esModuleInterop`?